src/export.h

```diff
@@ -0,0 +1,30 @@
+#ifndef EXPORT_H__
```

Comment:
This file really ought to be generated automatically using CMake's `generate_export_header()` function.